tests/drivers/i2c/i2c_slave_api/doc/index.rst

```diff
@@ -0,0 +1,55 @@
+.. _i2c_slave_api:
+
```

Comment:
Standard practice in the tests hierarchy is to put this sort of thing into `README.txt` at the top level of the application directory.  I suppose it could be `README.rst` too, but people aren't going to look for `doc/index.rst` to find this information.

Reply:
done